.autover/changes/8a5204e4-83ec-45ac-8c75-27512ae9cdf8.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.Tools",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Use MSBuild to find target framework"
```

Comment:
Add more explanation here for the significance of this for customers. Not sure everyone would know what this implies.